ibis/backends/tests/base.py

```diff
@@ -144,7 +144,7 @@ def geo(self) -> Optional[ir.TableExpr]:
 
     @property
     def api(self):
-        return getattr(ibis.backends, self.name()).Backend()
```

Comment:
this is for backwards compabtiblity?

Reply:
No, they are equivalent, since `ibis.sqlite` resolves via `__getattr__` and entrypoints to `ibis.backends.sqlite.Backend()`. The problem is that `ibis.omniscidb` resolves to the OmniSciDB backend (if installed), but this line resolves to `ibis.backends.omniscidb.Backend()`, which doesn't exist, since the backend is in another package (not in the `ibis/backends` directory  in the Ibis repo).